pkg/controller/user-cluster-controller-manager/kubevirt-vmi-eviction/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - sig-virtualization
+
+reviewers:
+  - sig-virtualization
+
+labels:
+  - sig/virtualization
+
+options:
+  no_parent_owners: true
```

Comment:
I think that file doesn't have to be changed :)

Reply:
? It's not changed, it's added. You think it should not be added ?

Reply:
ah my bad.. it should be added and please add a new line by the end of the file :)

Reply:
done